security.md

```diff
@@ -1,20 +1,40 @@
+
+For this purpose, OpenMRS app uses [BCrypt](https://en.wikipedia.org/wiki/Bcrypt "BCrypt") hashing function \(you can read more about password hashing [here](https://en.wikipedia.org/wiki/Cryptographic_hash_function#Password_verification "Password verification")\). BCrypt is currently one of the safest password hashing functions available and it is future-proof \(it's easy to increase its complexity in the future to prevent from [brute-force attacks](https://en.wikipedia.org/wiki/Brute-force_attack "Brute-force attack")\).
+
+##### 2. Local database encryption
+
+Managing the information about patients (personal data, medical record) is a huge responsibility. To protect the copy of the data on the phone, the database is encrypted using [AES-256 cipher](https://pl.wikipedia.org/wiki/Advanced_Encryption_Standard "Advanced Encryption Standard"), which uses a combination of user's username and password as its key. This encryption function makes it impossible to decrypt the data without entering valid username and password, so even if the device is stolen or breached - it's not possible to decrypt the database without login credentials.
+
+### Server side
+
+##### 1. Patient Identifiers are Validated
 
 Specifically, OpenMRS uses a variation of Luhn algorithm also known as the 'mod10' algorithm. Simply put, this algorithm has the capacity to validate a variety of identification numbers. In fact, this is commonly used to validate credit card numbers. For more information, please check out the following [link](https://www.geeksforgeeks.org/luhn-algorithm/ "Luhn Algorithm") regarding the Luhn algorithm to gain a better understanding on how it works.
 
-OpenMRS utilizes a variation of the Luhn algorithm which the Regenstrief Institue has created. In this variation, letters as well as numbers in the identifier (i.e., alphanumeric identifiers) are allowed. 
+OpenMRS utilizes a variation of the Luhn algorithm which the Regenstrief Institue has created. In this variation, letters as well as numbers in the identifier (i.e., alphanumeric identifiers) are allowed.
 
-This is especially useful when creating new Patient Identifiers. Manually inputting Patient Identifiers is very prone to errors, which is why a check digit algorithm is necessary. For more information, please check out the following [link](https://wiki.openmrs.org/display/docs/Check+Digit+Algorithm "OpenMRS Luhn Algorithm Documentation") 
-#### 2. Data is Transmitted Through a Secure SSL Conncection
+This is especially useful when creating new Patient Identifiers. Manually inputting Patient Identifiers is very prone to errors, which is why a check digit algorithm is necessary. For more information, please check out the following [link](https://wiki.openmrs.org/display/docs/Check+Digit+Algorithm "OpenMRS Luhn Algorithm Documentation")
+
+##### 2. Data is Transmitted Through a Secure SSL Conncection
```

Comment:
What is the change here?

Reply:
It's the formatting done by GitBook Editor. (it automatically removed a whitespace and separated the title from a paragraph - it has no visible effect but it's clearer to read)